catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -160,7 +165,7 @@
     public CachingFederationStrategy(ExecutorService queryExecutorService,
             List<PreFederatedQueryPlugin> preQuery, List<PostFederatedQueryPlugin> postQuery,
             SolrCache cache, ExecutorService cacheExecutorService, FilterAdapter adapter,
-            FilterBuilder builder) {
+            FilterBuilder builder, ValidationFilterFactory validationFilterFactory) {
```

Comment:
You haven't added this new argument to your blueprint. The implementation won't be started up successfully.